libs/blocks/fragment/fragment.js

```diff
@@ -1,5 +1,6 @@
 /* eslint-disable max-classes-per-file */
-import { createTag, getConfig, loadArea, localizeLink, customFetch } from '../../utils/utils.js';
+import { createTag, getConfig, loadArea, localizeLinkAsync, customFetch } from '../../utils/utils.js';
+import { getMepLingoContext, getQueryIndexPaths, fetchMepLingoThenFallback, fetchMepLingoParallel } from '../../features/mep/lingo.js';
```

Comment:
Do we 100% of the time need this import, or is there some condition we can check for first?

Often, unfortunately, authors will add a fragment in the first section of a page. If we do not have a conditional governing this import, then it will create a request chain for this file that will make its way in before LCP.